### PR TITLE
Added new prefix in "Adding a consumer to a consumer group"

### PR DESCRIPTION
**Pull request by fmcdonal**

### CODE REVIEW

File: docs/kafka/consumer-configuration-kafka/README.adoc [hunk 1]
```diff
@@ -139,10 +139,13 @@ sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule require
 == Adding a consumer to a consumer group
 
 [role="_abstract"]
+Consumer groups are created by specifying a consumer group ID in the user's client application, and not from the CLI or web console.
```
Comment (jbyrne-redhat) [inline docs/kafka/consumer-configuration-kafka/README.adoc:142]: Two things I suggest changing here:

 - Write in the active voice, not the passive voice.
 - Don't refer to "the user" in the third person. Our documentation is directed **to** the user. 
 
So, something like:
`To create a consumer group, you specify a consumer group ID in a client application. You do *not* create consumer groups in the {product-kafka} web console, or using the CLI.`

**Review by jbyrne-redhat — CHANGES_REQUESTED** [review]
A couple of suggested changes, @fmcdonal.

File: docs/kafka/consumer-configuration-kafka/README.adoc [hunk 2]
```diff
@@ -139,7 +139,7 @@ sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule require
 == Adding a consumer to a consumer group
 
 [role="_abstract"]
-Consumer groups are created by specifying a consumer group ID in the user's client application, and not from the CLI or web console.
+To create a consumer group you specify a consumer group ID in a client application. You do *not* create consumer groups in the {product-kafka} web console, or using the CLI.
```
Comment (mmurphy) [inline docs/kafka/consumer-configuration-kafka/README.adoc:142]: looks ok, but might be better changing "in a client application" to "in your consumer application"

Comment (fmcdonal) [reply docs/kafka/consumer-configuration-kafka/README.adoc:142]: appplied

**Review by jbyrne-redhat — APPROVED** [review]

**Pull request merged**